packages/hdwallet-native/src/cosmos.ts

```diff
@@ -11,7 +11,7 @@ import * as util from "./util";
 
 const ATOM_CHAIN = "cosmoshub-4";
 
-const protoTxBuilder = PLazy.from(() => import("@shapeshiftoss/proto-tx-builder"));
+const protoTxBuilder = PLazy.from(() => import("@keepkey/proto-tx-builder"));
```

Comment:
Revert. Except for hdwallet-keepkey, we don't import @keepkey namespace packages in hdwallet.